roles/foreman_installer/tasks/installer_version.yml

```diff
@@ -3,6 +3,10 @@
     src: /usr/share/foreman-installer/VERSION
   register: foreman_installer_version_file
 
+- name: 'Decode installer version'
+  set_fact:
+    foreman_installer_version: "{{ foreman_installer_version_file['content'] | b64decode }}"
```

Comment:
Can you also modify `upgrade.yml` to use this new variable rather than `b64decode` it manually?

Reply:
sure